src/index.js

```diff
@@ -1,5 +1,7 @@
 const path = require('path');
 
+const TEST_METHOD_CALL_NAMES = ['test', 'tests', 'module'];
```

Comment:
Where does the `tests` API come from?

Reply:
Discussed offline. Thanks Steve. Instead of searching against these hard coded call names, I'll take the approach we discussed.